src/pages/Rules/index.tsx

```diff
@@ -0,0 +1,51 @@
+import "./rules.css";
+import rules from "assets/hero/rules.png";
+import React from "react";
+import { FingerRight } from "assets/icons";
+import { Link } from "react-router-dom";
+
+export function Rules() {
+  return (
+    <section>
+      <div className="card">
+        <img src={rules} alt="rules__page--img" />
+        <h6>Rules to be followed</h6>
+        <ul>
+          <li>
+            <span>
+              <FingerRight className="icon-arrow" />
+            </span>
+            There will be 5 questions.
+          </li>
+          <li>
+            <span>
+              <FingerRight className="icon-arrow" />
+            </span>
+            Each question will be of one mark.
```

Comment:
Here, I think, you can map over an array as only the rules text is changing.
Something like this,
```javascript
["There will be  5 questions", "Each question will be of one mark", ...other rules].map((rule) => (

<li>
     <span>
              <FingerRight className="icon-arrow" />
            </span>
            {rule}
</li>

))
```

In this way, number of lines in the file will reduce.

Reply:
ok will make an array for storing rules